frontend/src/components/ThemedWindow.jsx

```diff
@@ -85,6 +87,11 @@ const useStyles = makeStyles(theme => ({
         borderImage: `url(${AVSRBorder}) 9 round`,
         border: '9px solid transparent',
       },
+      '&$noreset': {
+        borderImage: `none 9 round`,
```

Comment:
`borderImage: 'none'` is more semantically correct for that

Reply:
done